addons/web_tour/static/src/tour_service/tour_automatic.js

```diff
@@ -12,10 +12,6 @@ export class TourAutomatic {
     constructor(data) {
         Object.assign(this, data);
         this.steps = this.steps.map((step, index) => new TourStepAutomatic(step, this, index));
-        this.macroEngine = new MacroEngine({
```

Comment:
Correct me if I'm wrong, but it looks like this was the main (if not only) use case for the `MacroEngine` class. If that's the case, shouldn't you remove it as well, since it's pointless now?

Reply:
The goal is to definitely remove it yes ... But I want to minimize the diff in this PR.